pkgs/development/interpreters/wasmtime/default.nix

```
@@ -2,17 +2,22 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "wasmtime";
-  version = "0.39.1";
+  version = "0.40.0";
 
   src = fetchFromGitHub {
     owner = "bytecodealliance";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-cU03wm1+V++mV7j7VyMtjAYrPldzTysNzpJ8m0q4Rx8=";
+    sha256 = "sha256-i3dtjKy5YTyNkEaDcGxsrd4lbyUBd25OARESbJVK9uY=";
     fetchSubmodules = true;
   };
 
-  cargoSha256 = "sha256-DnThste0SbBdpGAUYhmwbdQFNEB3LozyDf0X8r2A90Q=";
+  cargoSha256 = "sha256-wlcYdZ8Akkpicyw74sh4CNwSstCtUJxf+oSEL/mUtKc=";
+
+  cargoBuildFlags = [ "--package" "wasmtime-cli"
+                      "--package" "wasmtime-c-api" ];
```

```suggestion
  cargoBuildFlags = [
    "--package wasmtime-cli"
    "--package wasmtime-c-api"
  ];
```
I am not sure if the splitting still works. If not we could just do:

```suggestion
  cargoBuildFlags = [
    "--package" "wasmtime-cli"
    "--package" "wasmtime-c-api"
  ];
```